docs/native-components-ios.md

```diff
@@ -234,15 +234,7 @@ render() {
 }
 ```
 
-Here you can see that the shape of the region is explicit in the JS documentation - ideally we could codegen some of this stuff, but that's not happening yet.
```

Comment:
The codegen comment I removed just because it seemed irrelevant to the topic, but I admit it's also irrelevant to the `requireNativeComponent` change so could bring it back if you wish